README.md

```diff
@@ -38,79 +32,409 @@ IncludeOS is free software, with "no warranties or restrictions of any kind".
 
 A longer list of features and limitations can be found on our [documentation site](http://includeos.readthedocs.io/en/latest/Features.html).
 
-## Getting started
+___
 
-### Set custom location and compiler
+### Getting started with IncludeOS development
 
-By default the project is installed to `/usr/local/includeos`.
+The [IncludeOS](https://www.includeos.org/) conan recipes are developed with [Conan version 1.13.1](https://github.com/conan-io/conan/releases/tag/1.13.1) or newer.
 
-However, it is recommended to choose a custom location as well as select the compiler we want clang to find. In this document we assume you install IncludeOS in your home directory, in the folder `~/includeos`.
+For Mac OS ensure that you have a working installation of [brew](https://brew.sh/) to be able to install all dependencies.
 
-To do this we can edit `~/.bash_profile` (mac os) or `~/.bashrc` (linux), adding these lines at the end of the file:
 
```

Comment:
Could we have an index with links to the main headlines somewhere here?

Reply:
Added reference labels and Content at the beginning.